conf/defaults.ini

```diff
@@ -885,6 +885,7 @@ app_tls_skip_verify_insecure = false
 allow_loading_unsigned_plugins =
 # Enable or disable installing plugins directly from within Grafana.
 plugin_admin_enabled = false
+plugin_admin_remote_manage_enabled = false
```

Comment:
Nit. For the external user management, https://grafana.com/docs/grafana/latest/administration/configuration/#external-user-management, we use external. Here we  use remote. Do we want to make this consistent?

Reply:
Good point. I'm easy either way. How about you @bergquist?

Reply:
No strong opinion but consistency is always nice :)

Reply:
Done 😄